pkg/kubelet/kubeletconfig/util/files/files.go

```diff
@@ -91,7 +91,7 @@ func WriteTmpFile(fs utilfs.Filesystem, path string, data []byte) (tmpPath strin
 		}
 	}()
 
-	// Name() will be an absolute path when using utilfs.DefaultFS, because ioutil.TempFile passes
+	// Name() will be an absolute path when using utilfs.DefaultFS, because os.CreateTemp passes
```

Comment:
Here is using fs.TempFile()

https://github.com/kubernetes/kubernetes/blob/8c6e8ecb43b2674390f59bef72659fa0c7f529cf/pkg/util/filesystem/defaultfs.go#L107-L114

This code is not changed in this PR and is still using ioutil.TempFile.